routellm/routers/routers.py

```diff
@@ -217,9 +217,10 @@ def __init__(
         weak_model="mixtral-8x7b-instruct-v0.1",
         hidden_size=128,
         num_models=64,
-        text_dim=1536,
+        text_dim=768,
         num_classes=1,
         use_proj=True,
+        embedding_model="all-mpnet-base-v2",
```

Comment:
Maybe better to use OpenAI's embeddings by default to preserve our existing behavior? I'll add some docs to describe the different options here.